StRoot/macros/embedding/anapicodst/README

```diff
@@ -0,0 +1,7 @@
+how to run:
+1) convert embedding MuDst.root to picoDst.root
+adjust the pico chain setting in genDst.csh, then
+./genDst.csh a.MuDst.root
+Note that only the pico code (StPicoDstMaker/StPicoEvent) in recent STAR library (>= SL21b) includes MC information. One should use the same library (or pico code) to convert real data MuDst and embedding MuDst.
+2) run the example analysis code (SL19b is used here for testing only, StRoot/StPicoDstMaker and StRoot/StPicoEvent should be checked out from git and cons-ed in SL19b for this test)
```

Comment:
Just a suggestion: I imagine this is a good place to offer a suggestion on how to have local compilations of just these Pico codes, as people running jobs probably don't want to have the SL21b-or-later library copied into each job.

Reply:
Hi Gene, this script is only used for testing the new pico code. I somehow use ACLIC to compile the makePicoDstQA.C, which only works on old libraries like SL19b, the newer library (SL19e, for example) breaks ACLIC.